### DISCUSSION TITLE
Hi can we have an option to set a hotkey to lock a tab please? (on windows chrome)

### DISCUSSION THREAD

**Discussion by Slyger** [tabwrangler/tabwrangler#481] [General]
Hi can we have an option to set a hotkey to lock a tab please? (on windows chrome)

**Comment by ssorallen** [comment]
This already exists! In Chrome do the following:

1. Go to "Settings"
2. Select "Extensions" at the bottom of the left nav
3. Select "Keyboard shortcuts" in the left nav
4. Choose a shortcut key for "Lock/Unlock the active tab"
<img width="683" alt="Screenshot 2024-07-24 at 9 49 05 AM" src="https://github.com/user-attachments/assets/a14b07b3-03eb-41eb-9e95-1ccfa36e0dbd">
